backend/src/main/java/com/bakdata/conquery/models/events/parser/Parser.java

```diff
@@ -91,9 +85,18 @@ public MAJOR_JAVA_TYPE addLine(MAJOR_JAVA_TYPE v) {
 		return v;
 	}
 
+	/**
+	 * Register/Analyze an incoming value for {@link #decideType()}.
+	 */
+	protected void registerValue(MAJOR_JAVA_TYPE v) {
+	}
```

Comment:
Warum ist die standardmäßig leer?

Reply:
Sollte das nicht in den meisten fällen etwas tun?

Reply:
Ich glaube es gibt ein paar Parser die das nicht benutzen